jekyll/_includes/utility-icons/external.svg

```diff
@@ -1 +1,7 @@
-<svg fill="none" height="12" viewBox="0 0 12 12" width="12" xmlns="http://www.w3.org/2000/svg"><path d="m9.33318 10.6667h-8v-8.00003h3.33333v-1.33334h-3.706665c-.26-.00666-.506666.08667-.686666.26667-.186667.17333-.28000037.42-.2733337.67333v8.79337c-.00666667.2533.0866667.4933.2733337.6733.18.18.433333.2733.686666.2667h8.746665c.53329 0 .95999-.42.95999-.94v-3.73337h-1.33332zm.39333-9.33337h-2.39333c-.36667 0-.66667-.3-.66667-.666663 0-.366667.3-.666667.66667-.666667h4.00002c.3666 0 .6666.3.6666.666667v4.000003c0 .36666-.3.66666-.6666.66666-.3667 0-.6667-.3-.6667-.66666v-2.39334l-2.74665 2.74667c-.26.26-.68.26-.94 0s-.26-.68 0-.94z" fill="currentColor"/></svg>
\ No newline at end of file
+{% if include.size %}
+  {% assign size = include.size %}
+{% else %}
+  {% assign size = 12 %}
+{% endif %}
+
+<svg viewBox="0 0 {{size}} {{size}}" fill="currentColor" xmlns="http://www.w3.org/2000/svg" class="css-bleycz"><path viewBox="0 0 {{size}} {{size}}" fill-rule="evenodd" d="M17,19 L5,19 L5,7 L10,7 L10,5.00083798 L4.44085883,5.00083798 C4.05495161,4.9876138 3.68075435,5.13139282 3.4077085,5.39780868 C3.13466265,5.66422454 2.98730555,6.02933568 3.00085883,6.40587245 L3.00085883,19.5941275 C2.98730555,19.9706643 3.13466265,20.3357755 3.4077085,20.6021913 C3.68075435,20.8686072 4.05495161,21.0123862 4.44085883,20.999162 L17.56,20.999162 C18.35529,20.999162 19,20.3701067 19,19.5941275 L19,14 L17,14 L17,19 Z M17.5857864,5 L14,5 C13.4477153,5 13,4.55228475 13,4 C13,3.44771525 13.4477153,3 14,3 L20,3 C20.5522847,3 21,3.44771525 21,4 L21,10 C21,10.5522847 20.5522847,11 20,11 C19.4477153,11 19,10.5522847 19,10 L19,6.41421356 L14.8786797,10.5355339 C14.4881554,10.9260582 13.8549904,10.9260582 13.4644661,10.5355339 C13.0739418,10.1450096 13.0739418,9.51184464 13.4644661,9.12132034 L17.5857864,5 Z"></path></svg>
```

Comment:
This icon flashes a different size when the page load:

![Kapture 2021-10-01 at 19 26 38](https://user-images.githubusercontent.com/1449325/135700979-77dc5d49-dfd8-4acf-bf94-2a9fbdaea456.gif)

Can you please fix this before our pairing? 😊

Reply:
Done!